roles/cyclictest/templates/cyclictestjob.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       runtimeClassName: "{{ workload_args.runtime_class }}"
 {% endif %}
       ttlSecondsAfterFinished: 600
-{% if workload_args.nodeselector is defined %}
+{% if workload_args.node_selector is defined %}
```

Comment:
It's odd because it looks like it's already `node_selector` in `master`?
https://github.com/cloud-bulldozer/benchmark-operator/blob/master/roles/cyclictest/templates/cyclictestjob.yaml#L23

Reply:
me-- I forgot to rebase.

Reply:
ack, rebase and we can move it on